Experimental/DistributedAuthoritySample/Assets/Scripts/Input/AvatarInputs.cs

```diff
@@ -1,7 +1,9 @@
+using System.Runtime.CompilerServices;
 using UnityEngine;
 using UnityEngine.InputSystem;
 
-namespace com.unity.multiplayer.samples.distributed_authority.input
+[assembly: InternalsVisibleTo("com.unity.multiplayer.samples.socialhub.player")]
```

Comment:
considering that the AvatarInputs class might go away at any moment in the future, I'd recommend to put the `InternalsVisibleTo` into an AssmeblyInfo.cs file like we do in the Small Scale Template. This centralizes the place where all internals are "shared", and makes it independent from individual script classes: https://github.cds.internal.unity3d.com/unity/com.unity.template.multiplayer.ngo/blob/main/Assets/Scripts/Runtime/AssemblyInfo.cs

Reply:
That's neat! Done.